kernel/tests/read.rs

```diff
@@ -522,32 +527,36 @@ fn read_table_data(
 ) -> Result<(), Box<dyn std::error::Error>> {
     let path = std::fs::canonicalize(PathBuf::from(path))?;
     let url = url::Url::from_directory_path(path).unwrap();
-    let engine = DefaultEngine::try_new(
+    let default_engine = DefaultEngine::try_new(
         &url,
         std::iter::empty::<(&str, &str)>(),
         Arc::new(TokioBackgroundExecutor::new()),
     )?;
+    let sync_engine = delta_kernel::engine::sync::SyncEngine::new();
+
+    let engines: &[&dyn Engine] = &[&sync_engine, &default_engine];
+    for &engine in engines {
+        let table = Table::new(url.clone());
+        let snapshot = table.snapshot(engine, None)?;
+
+        let read_schema = select_cols.map(|select_cols| {
+            let table_schema = snapshot.schema();
+            let selected_fields = select_cols
+                .iter()
+                .map(|col| table_schema.field(col).cloned().unwrap())
+                .collect();
+            Arc::new(Schema::new(selected_fields))
```

Comment:
This is just an indentation change. Review with whitespace changes hidden if you're seeing this comment:
![image](https://github.com/user-attachments/assets/401077cd-2876-4ceb-b42a-64890cd6bc95)